homeassistant/components/bouncie/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""The bouncie integration."""
+from __future__ import annotations
+
+import datetime
+import logging
+
+from aiohttp import ClientSession
+from aiohttp.client_exceptions import ClientConnectorError
+from bounciepy import AsyncRESTAPIClient
+from bounciepy.exceptions import BouncieException, UnauthorizedError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+# For your initial PR, limit it to 1 platform.
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up bouncie from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+    update_interval = datetime.timedelta(seconds=60)
+    session = async_get_clientsession(hass=hass)
```

Comment:
you don't need to create variables for them, fill the coordinator parameters directly